### ISSUE TITLE
MP: Cannot clone listing

### ISSUE THREAD

**Issue by DrDBanner** [particl/particl-desktop#1499]
# Describe bug/issue

Cannot clone listing. -> Popup: "Could not create the ListingItemTemplate!"

marketplace.log output 
```
- debug: [api/controllers/RpcController] controller.handleRPC(): 0=[template add,1,TOKEN2049 Ticket,Get a Ticket while it is hot,Buy it and burn it. ,5,SALE,PARTICL,5,1,2]
2019-03-10T13:59:08.329Z - error: [api/repositories/ListingItemTemplateRepository] error:  0=[errno=19, code=SQLITE_CONSTRAINT]
2019-03-10T13:59:08.330Z - info: [app] [0mPOST /api/rpc [33m400 [0m2.321 ms - 156[0m
```
..

# How to reproduce

- Go to "Sell" 
- Expand published listing
- Hit "clone listing"
- Hit "Save and Publish" 

# More information

- Operating system and version: MacOS 10.14.3
- Client version: 2.0.0-alpha-testnet

**Comment by DrDBanner** [comment]
If any information within the item is changed before saving it works without problems. Is it possible that the "title" is a key and therefore unique?

**Comment by zaSmilingIdiot** [comment]
The entire textual content of the listing is used to determine uniqueness of the listing. We are aware of this internally and have discussed a potential fix for it (its on the list of things to be implemented), but thanks for raising this - we'll get it addressed.

**Comment by zaSmilingIdiot** [comment]
This has been resolved and can be tested in the latest build: v2.0.0-alpha1-testnet
